core/src/Platforms/android/CoreCryptographyHelpers.cs

```diff
@@ -28,12 +28,12 @@
 using System;
 using System.Security.Cryptography;
 using System.Text;
-using Microsoft.Identity.Client.Internal;
+using Microsoft.Identity.Core.Helpers;
 
-namespace Microsoft.Identity.Client
+namespace Microsoft.Identity.Core
 {
     [Android.Runtime.Preserve(AllMembers = true)]
-    internal class CryptographyHelper
+    internal class CoreCryptographyHelpers
```

Comment:
>Core [](start = 19, length = 4)

remove Core ?

Reply:
we already have cryptographyhelper class in adal/msal. it would be confusing to use class with same name in 2 different namespaces.

---
In reply to: [161068014](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161068014) [](ancestors = 161068014)

Reply:
pls. keep as is

---
In reply to: [161134996](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/941#discussion_r161134996) [](ancestors = 161134996,161068014)